dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -5945,7 +6075,7 @@ public Set<String> getInstalledAppNames(String... possiblyInstalledAppNames) thr
             for (String name : possiblyInstalledAppNames)
                 counters.put(name, 0);
 
-            for (String line : findStringsInFileInLibertyServerRoot(".*((CWWKZ0)|(J2CA7))00[139]I: .*", "logs/messages.log"))
+            for (String line : findStringsInLogs(".*((CWWKZ0)|(J2CA7))00[139]I: .*"))
```

Comment:
I _think_ this change is right - those two methods are not the same but I think they do the same thing.

Reply:
Yeah, apparently it is possible to have the logs somewhere other than the server root. The original method does not check for that.